IndicoV2.Abstractions/Submissions/Models/Submission.cs

```diff
@@ -70,14 +70,29 @@ public SubmissionSs(IListSubmissions_Submissions_Submissions submission)
             Errors = _ssSubmission.Errors;
         }
 
+        private string ParseStatus(StrawberryShake.SubmissionStatus? status)
+        {
+            switch (status)
+            {
+                case StrawberryShake.SubmissionStatus.Processing : return "PROCESSING";
```

Comment:
You can just return the matching SubmissionStatus enum here, rather than using a string and parsing again :). For the default, you could raise an exception (maybe invalid argument) that explains that we got an unexpected status, etc.